src/index.js

```diff
@@ -1,17 +1,6 @@
 import React from 'react';
-import ReactDOM from 'react-dom/client';
+import ReactDOM from 'react-dom';
+import Calculator from './components/calculator';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
 
-const root = ReactDOM.createRoot(document.getElementById('root'));
-root.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-);
-
-// If you want to start measuring performance in your app, pass a function
-// to log results (for example: reportWebVitals(console.log))
-// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
-reportWebVitals();
+ReactDOM.render(<Calculator />, document.getElementById('root'));
```

Comment:
- [ ] [optional] It would have been better to change the renderer to the latest rendered supported by react version 18 which you are currently using.
![image](https://user-images.githubusercontent.com/61408860/189150274-8ba3d419-ca09-4c11-a366-9349f1c8ca57.png)